From acc753860b87d3c96d01963b5ea259695ee9f683 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Thu, 4 May 2017 12:18:33 +0100
Subject: [PATCH] Related: rhbz#1447830 ExportHiddenSlides isn't persistent in
 dialog

every time the dialog is run its back to its default setting again
unlike the other options

Change-Id: Ieb6c94a4ec8670957f9e295acee7114d5c4c6d5a
(cherry picked from commit 651169f85640c9d15381c424dfe0ed5cd93e58ee)
---
 officecfg/registry/schema/org/openoffice/Office/Common.xcs | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/officecfg/registry/schema/org/openoffice/Office/Common.xcs b/officecfg/registry/schema/org/openoffice/Office/Common.xcs
index 32c3cd7..973c02e 100644
--- a/officecfg/registry/schema/org/openoffice/Office/Common.xcs
+++ b/officecfg/registry/schema/org/openoffice/Office/Common.xcs
@@ -5168,6 +5168,13 @@
             </info>
             <value>true</value>
           </prop>
+          <prop oor:name="ExportHiddenSlides" oor:type="xs:boolean" oor:nillable="false">
+            <info>
+              <desc>Specifies if hidden slides are exported to PDF. (Hidden slides
+              are available in Impress documents only).</desc>
+            </info>
+            <value>false</value>
+          </prop>
           <prop oor:name="ExportFormFields" oor:type="xs:boolean" oor:nillable="false">
             <info>
               <desc>Specifies whether form fields are exported as widgets or
-- 
2.9.3

